src/structure_managers/structure_manager.hh

```diff
@@ -144,21 +144,11 @@ namespace rascal {
     //! Overload to build the tuple
     template <typename... PropertyTypes, typename Manager>
     struct ClusterIndexConstructor<std::tuple<PropertyTypes...>, Manager> {
-      static inline decltype(auto) make(Manager & manager) {
+      static std::tuple<PropertyTypes...> make(Manager & manager) {
```

Comment:
I would keep auto in this case since the output type is quite clear no ?

Reply:
I like to keep the real types if it is possible. It is quite clear, but it still saves brain power when it is written out.